app/models/date_attended.rb

```diff
@@ -12,9 +12,19 @@
 
 class DateAttended < ActiveRecord::Base
   belongs_to :fee
+  belongs_to :expense
```

Comment:
I really think this ought to be a polymorphic association. Replacing both belongs to associations with:

belongs_to :date_attendable